webrtc.html

```diff
@@ -8560,6 +8562,11 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <p>The event type of this event handler is
               <code><a>tonechange</a></code>.</p>
             </dd>
+            <dt><dfn><code>canInsertDTMF</code></dfn> of type <span class=
+            "idlAttrType"><a>boolean</a></span>, readonly</dt>
+            <dd>
+              <p>Whether the <a>RTCDTMFSender</a> is capable of sending DTMF.</p>
```

Comment:
Those clarifications are intended for PR https://github.com/w3c/webrtc-pc/pull/1542